src/pages/Home/Home.layout.jsx

```diff
@@ -16,13 +17,24 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
  */
 const Home = ({ title, subtitle, buttonText }) => (
   <Header>
-    <Logo src={logo} alt="logo" />
-    <Text as="h1" color="secondary" large>
-      {title}
-    </Text>
-    <Text small>{subtitle}</Text>
-    <ManImage src={deliveryMan} alt="Delivery man" />
-    <Button to="/check">{buttonText}</Button>
+    <HalfSection bg={Colors.yellow06}>
+      {/**/}
+      <Logo src={logo} alt="logo" />
+      <ManImage style={{ display: 'nones' }} src={deliveryMan} alt="Delivery man" />
+      {/**/}
+    </HalfSection>
+
+    <HalfSection mobileHeight="400px" bg={Colors.blue08} fxJtyCt="center" fxAlIt="center">
```

Comment:
The props names can be better, please change them, currently is hard to follow that they meant, even after looking at the CSS component

Reply:
Also there are not other places these props changes, can you try using defaults?

Reply:
props names changed